docs/source/executing-operations/subscription-callback-protocol.mdx

```diff
@@ -3,15 +3,13 @@ title: HTTP callback protocol for GraphQL subscriptions
 description: For federated subgraphs communicating with the Apollo Router
 ---
 
-> This protocol is currently in [preview](/resources/product-launch-stages#preview). Breaking changes might be introduced during the preview period.
->
-> **You only need to read this reference if you're adding support for this protocol to a GraphQL server library or other system!**
+<PreviewFeature discordLink="https://discord.com/channels/1022972389463687228/1148623262104965120"/>
```

Comment:
This Discord link is for a conversation for authorization directives in particular. If there's no specific channel/conversation to link, you can just leave the discord link blank and it'll default to the generic Apollo Discord.

```suggestion
<PreviewFeature />
```